test/views/html/incomeTaxComparison/MainSpec.scala

```diff
@@ -119,12 +176,34 @@ class MainSpec extends TaiViewSpec {
 
   private lazy val currentYearItem = EstimatedIncomeTaxComparisonItem(TaxYear(), 100)
   private lazy val startOfNextTaxYear = Dates.formatDate(TaxYear().next.start)
-  private lazy val nextYearItem = EstimatedIncomeTaxComparisonItem(TaxYear().next, 200)
-  private lazy val estimatedIncomeTaxComparisonViewModel = EstimatedIncomeTaxComparisonViewModel(Seq(currentYearItem, nextYearItem))
+  private lazy val nextYearItemMore = EstimatedIncomeTaxComparisonItem(TaxYear().next, 101)
+  private lazy val nextYearItemSame = EstimatedIncomeTaxComparisonItem(TaxYear().next, 100)
+  private lazy val nextYearItemLess = EstimatedIncomeTaxComparisonItem(TaxYear().next, 99)
+
+  private def estimatedIncomeTaxComparisonViewModel(nextYearValue: String) = {
```

Comment:
I'm not sure this function really adds much. What might be more useful is a function for building `IncomeTaxComparisonViewModel`'s. Something like:
```
private def buildIncomeTaxComparisonViewModel(currentYearItem: EstimatedIncomeTaxComparisonItem, nextYearItem: EstimatedIncomeTaxComparisonItem) = {
  IncomeTaxComparisonViewModel(
    "USERNAME",
    EstimatedIncomeTaxComparisonViewModel(Seq(currentYearItem, nextYearItem)),
    TaxCodeComparisonViewModel(Nil),
    TaxFreeAmountComparisonViewModel(Nil, Nil),
    IncomeSourceComparisonViewModel(Nil, Nil),
    false
  )
}
```
Usage would then be
```
private lazy val incomeTaxComparisonViewModelMore =
  buildIncomeTaxComparisonViewModel(currentYearItem, nextYearItemMore)
```